### DISCUSSION TITLE
sacli command?

### DISCUSSION THREAD

**Discussion by matt-harbers** [angristan/openvpn-install#928] [Help]
I'm trying to pull a list of users with permissions for our audit, and it seems like standard openvpn install has a tool called sacli that would provide this functionality.

Does anyone know if this utility is included and where it is?

**Comment by TinCanTech** [comment]
The OpenVPN tool `sacli` is for The OpenVPN Access Server commercial product.

The version of OpenVPN used in this repo is the free open source version, which has no such tool.

**Reply by matt-harbers** [reply]
Cool, is there any kind of way (even through config files) to see users and their permissions? Thanks in advance!
